src/lib/containers/row_renderers/utils/Icon.tsx

```diff
@@ -42,6 +42,7 @@ import {
 } from '../../../utils/SynapseConstants'
 
 import { ReactComponent as Data2Svg } from '../../../assets/icons/Data2.svg'
+import { ReactComponent as DatasetSvg } from '../../../assets/icons/Dataset.svg'
```

Comment:
I think this exists in MUI -- see '@material-ui/icons/GridOnTwoTone'. Can we use those in this Icon component?